lib/tasks/wikirate.rake

```diff
@@ -16,136 +16,68 @@ namespace :wikirate do
       mysql_args += " -p #{pwd}" if pwd
       system "mysql #{mysql_args} #{test_database} < #{db_path}"
     end
+    desc 'add wikirate test data to test database'
+    task :add_wikirate_test_data do
```

Comment:
vast improvement.  still seems like a pretty long task; would be nice to separate the dumping into its own task (eg if there were a testing problem you could try migrating an existing database and then dumping)